packages/components/calendar/src/use-scroll-end-callback.ts

```diff
@@ -0,0 +1,69 @@
+import {useCallback, useEffect, useRef} from "react";
+
+function useScrollEndCallback(debounce: number) {
```

Comment:
The value is not being synced during mouse scroll. Please fix it.

Reply:
Done.

Reply:
Could you please specify the tests you need? Given that original component had tests that failed to catch this bug, it's a somewhat open-ended task. My current codebase uses with a different UI library, so it's difficult to justify spending any more time on the NextUI repo at the moment, but if you can ring-fence exactly what needs doing I will make time for that.